RetrospectiveExtension.Backend/appsettings.json

```diff
@@ -13,6 +13,9 @@
     }
   },
   "ApplicationInsights": {
-    "InstrumentationKey":""
+    "InstrumentationKey":"",
+    "LogLevel":{
+      "Default": "Information"
```

Comment:
This will be fairly verbose. Should the level be different for the appsettings and appsettings.Development?

Reply:
good point. I will change it to `Info` for dev and `Warn` for Production